cmd/kola/spawn.go

```diff
@@ -174,7 +174,7 @@ func doSpawn(cmd *cobra.Command, args []string) error {
 				return fmt.Errorf("Could not read machine options: %v", err)
 			}
 
-			var machineOpts qemu.MachineOptions
+			var machineOpts platform.MachineOptions
```

Comment:
I think we should find a better home for this than `platform`, though not sure if that's worth blocking this PR.

Reply:
If I recall correctly I originally had it in a `platform/qemu` sub-package but I don't remember why I changed it back to just being in `platform`. I wouldn't personally be opposed to either having it in a sub-package or leaving it inside of `platform`.

@bgilbert did you have a particularly strong opinion either way?

Reply:
`platform` seems like the obvious place for it.  I don't think it's important to start a subpackage just for these structs.  I'm okay with other approaches though.

Reply:
I'll leave it in it's current place for this PR and we can decide later if we want to move it.